modules/custom/sql/lsb_mascot.sql

```diff
@@ -1,2 +1,7 @@
 -- Enables the LandSandBoat
-UPDATE npc_list SET name=CHAR(4), flag=1 WHERE name=CHAR(10);
+UPDATE npc_list
```

Comment:
The formatting here is awful, is there any way to make it _not_ split simple statements in small files over multiple lines? This sort of formatting would be great for big and complex queries, but we don't really keep those in files (apart from triggers.sql)